generator/internal/service_code_generator.cc

```diff
@@ -76,7 +76,7 @@ Status ServiceCodeGenerator::OpenNamespaces(Printer& p, NamespaceType ns_type) {
   }
   namespaces_ = BuildNamespaces(service_vars_["product_path"], ns_type);
   for (auto const& nspace : namespaces_) {
-    if (absl::EndsWith(nspace, "_CLIENT_NS")) {
+    if (nspace == "GOOGLE_CLOUD_CPP_NS") {
       p.Print("inline namespace $namespace$ {\n", "namespace", nspace);
     } else {
       p.Print("namespace $namespace$ {\n", "namespace", nspace);
```

Comment:
Nit: This is another one of those cases where it is really just the `"inline "` bit that is conditional.  That is, ...
```
  if (nspace == "GOOGLE_CLOUD_CPP_NS") p.Print("inline ");
  p.Print("namespace $namespace$ {\n", "namespace", nspace);
```